src/components/sections/NewsRoomSection.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import NewsRoom from "../NewsRoom";
+
+const NewsRoomSection = props => (
+  <div className="dg_section dark">
```

Comment:
change this to use the Section component from dotgov-components.

```js
import { Button, Section }  from "@baltimorecounty/dotgov-components";

```

```jsx
const ctaButton = <Button text="View all news" className="dg_button-secondary" />;
<Section className="dark" ctaButton={ctaButton}>
  // Your existing children go here
</Section>
```

Reply:
Example seen here, I updated most popular services to use this pattern without the button.

![image](https://user-images.githubusercontent.com/1933400/64988089-ab2d1d80-d898-11e9-883c-a44d3ecb48a3.png)

Reply:
done